### PR TITLE
NOISSUE - Update documentation

### PR DESCRIPTION
**Pull request by SammyOina**
To be merged after #18

### CODE REVIEW

File: manager/README.md [hunk 1]
```diff
@@ -41,9 +412,108 @@ make manager
 make install
 
 # set the environment variables and run the service
-./build/cocos-manager
+MANAGER_LOG_LEVEL=debug MANAGER_AGENT_GRPC_URL=localhost:7002 MANAGER_QEMU_USE_SUDO=false MANAGER_QEMU_ENABLE_SEV=false  ./build/cocos-manager
```
Comment (rodneyosodo) [inline manager/README.md:415]: Wrap them

File: manager/README.md [hunk 2]
```diff
@@ -41,9 +412,108 @@ make manager
...
 ```
 
+To enable [AMD SEV](https://www.amd.com/en/developer/sev.html) support, start manager like this 
+
+```sh
+MANAGER_LOG_LEVEL=debug MANAGER_AGENT_GRPC_URL=192.168.122.251:7002 MANAGER_QEMU_USE_SUDO=true MANAGER_QEMU_ENABLE_SEV=true MANAGER_QEMU_SEV_CBITPOS=51 ./build/cocos-manager
```
Comment (rodneyosodo) [inline manager/README.md:421]: same

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: manager/README.md [hunk 3]
```diff
@@ -25,8 +25,378 @@ The service is configured using the environment variables from the following tab
 | MANAGER_INSTANCE_ID      | Manager service instance ID                              |                                   |
 | COCOS_MESSAGE_BROKER_URL | message broker url                                       | nats://localhost:4222             |
 
+## Setup
+
+```sh
+git clone https://github.com/ultravioletrs/cocos-ai
```
Comment (drasko) [inline manager/README.md:31]: We changed this to `cocos`

**Review by drasko — CHANGES_REQUESTED** [review]

File: manager/README.md [hunk 4]
```diff
@@ -25,8 +25,378 @@ The service is configured using the environment variables from the following tab
...
+mkdir tmp
+
+FOCAL=focal-server-cloudimg-amd64.img
+wget -O img/$FOCAL https://cloud-images.ubuntu.com/focal/current/$FOCAL
+# focal-server-cloudimg-amd64 comes without the root password
+sudo apt-get install libguestfs-tools
+PASSWORD=coolpass
+sudo virt-customize -a img/$FOCAL --root-password password:$PASSWORD
+```
+#### Resize disk image, partition and filesystem
```
Comment (danko-miladinovic) [inline manager/README.md:75]: Can you please explain why we need to resize the partition and filesystem? Do we need to do it in order to have enough space for golang and agent.

File: manager/README.md [hunk 5]
```diff
@@ -25,8 +25,378 @@ The service is configured using the environment variables from the following tab
...
+
+# Update the PATH environment variable to include the directory containing the Go binary.
+export PATH=$PATH:/usr/local/go/bin
+
+# Verify the installed Go version by running the "go version" command.
+go version
+# => # go version go1.21.4 linux/amd64
+```
+
+#### Build and run Agent
```
Comment (danko-miladinovic) [inline manager/README.md:216]: Can you please add a short section before this one (`Build and run Agent`) where you give commands on how to start NATS?

File: manager/README.md [hunk 6]
```diff
@@ -25,8 +25,378 @@ The service is configured using the environment variables from the following tab
...
+
+```sh
+git clone https://github.com/ultravioletrs/cocos
+cd cocos
+
+# Build the 'agent' executable and save it to the 'build' directory.
+go build -o build/agent cmd/agent/main.go
+
+# Start the 'agent' executable in the background using '&' at the end.
+./build/agent &
```
Comment (danko-miladinovic) [inline manager/README.md:226]: After you have added the NATS section, can please explain how to start the `agent` with NATS server?

File: manager/README.md (hunk 4 above)
Comment (SammyOina) [reply manager/README.md:75]: Yes we need some additional space for agent golang dependencies before we build agent

File: init/systemd/cocos-agent.service [hunk 7]
```diff
@@ -10,6 +10,7 @@ Environment=NETWORK_INTERFACE=enp0s3
 Environment=AGENT_GRPC_HOST=10.0.2.15
 Environment=AGENT_GRPC_PORT=7002
 Environment=AGENT_LOG_LEVEL=info
+Environment=COCOS_MESSAGE_BROKER_URL=192.168.100.4:4222
```
Comment (drasko) [inline init/systemd/cocos-agent.service:13]: What is this and why is this IP addr chosen?

Comment (SammyOina) [reply init/systemd/cocos-agent.service:13]: It is the host ip, this is specific in my case and could change for different users

**Comment by drasko** [comment]
@SammyOina please update this one according to the new changes

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**